core/src/test/scala/unit/kafka/server/CreateTopicsRequestTest.scala

```diff
@@ -119,54 +107,22 @@ class CreateTopicsRequestTest extends AbstractCreateTopicsRequestTest {
 
   @Test
   def testInvalidCreateTopicsRequests() {
-    // Duplicate
-    val singleRequest = new CreateTopicsRequest.Builder(Map("duplicate-topic" ->
-        new CreateTopicsRequest.TopicDetails(1, 1.toShort)).asJava, 1000).build()
-    validateErrorCreateTopicsRequests(singleRequest, Map("duplicate-topic" -> error(Errors.INVALID_REQUEST,
-      Some("""Create topics request from client `client-id` contains multiple entries for the following topics: duplicate-topic"""))),
-      requestStruct = Some(toStructWithDuplicateFirstTopic(singleRequest)))
-
-    // Duplicate Partial with validateOnly
-    val doubleRequestValidateOnly = new CreateTopicsRequest.Builder(Map(
-      "duplicate-topic" -> new CreateTopicsRequest.TopicDetails(1, 1.toShort),
-      "other-topic" -> new CreateTopicsRequest.TopicDetails(1, 1.toShort)).asJava, 1000, true).build()
-    validateErrorCreateTopicsRequests(doubleRequestValidateOnly, Map(
-      "duplicate-topic" -> error(Errors.INVALID_REQUEST),
-      "other-topic" -> error(Errors.NONE)), checkErrorMessage = false,
-      requestStruct = Some(toStructWithDuplicateFirstTopic(doubleRequestValidateOnly)))
-
-    // Duplicate Partial
-    val doubleRequest = new CreateTopicsRequest.Builder(Map(
-      "duplicate-topic" -> new CreateTopicsRequest.TopicDetails(1, 1.toShort),
-      "other-topic" -> new CreateTopicsRequest.TopicDetails(1, 1.toShort)).asJava, 1000).build()
-    validateErrorCreateTopicsRequests(doubleRequest, Map(
-      "duplicate-topic" -> error(Errors.INVALID_REQUEST),
-      "other-topic" -> error(Errors.NONE)), checkErrorMessage = false,
-      requestStruct = Some(toStructWithDuplicateFirstTopic(doubleRequest)))
-
     // Partitions/ReplicationFactor and ReplicaAssignment
-    val assignments = replicaAssignmentToJava(Map(0 -> List(0)))
-    val assignmentRequest = new CreateTopicsRequest.Builder(Map("bad-args-topic" ->
-        new CreateTopicsRequest.TopicDetails(assignments)).asJava, 1000).build()
```

Comment:
Should we keep the 1000 timeout?

Reply:
See above, let's set it in `CreateTopicsRequest.json`